mmcv/cnn/utils/flops_counter.py

```diff
@@ -503,8 +504,8 @@ def batch_counter_hook(module, input, output):
         batch_size = len(input)
     else:
         pass
-        print('Warning! No positional inputs found for a module, '
-              'assuming batch size is 1.')
+        warnings.warn('No positional inputs found for a module, '
+                      'assuming batch size is 1.')
```

Comment:
The above line is `pass` so will the warning be skipped?

Reply:
No, `pass` is only a placeholder statement. Actually, I do not know why there is a `pass`.

Reply:
OK, we can delete the pass

Reply:
done